_examples/maps/test.py

```diff
@@ -7,11 +7,47 @@
 
 a = maps.New()
 b = {1: 3.0, 2: 5.0}
+
+# set to true to test all the builtin map functionality
+# due to the random nature of map access in Go, this will
+# cause go test to fail randomly so it is off by default
+# but should be re-enabled when anything significant is
+# changed!
+testall = False
```

Comment:
we could sort the lists returned by `keys()` and `values()`.
so we don't have to enable/disable this.